tests/test_special_ops.py

```diff
@@ -343,6 +344,25 @@ def test_accuracy_unique(shape, dtype, sorted, return_inverse, return_counts):
                 return_counts=return_counts,
             )
             assert res_out.numel() == ref_out.numel()
+
+
+@pytest.mark.parametrize("shape", VSTACK_SHAPES)
```

Comment:
Please test this 1D case:

```
@pytest.mark.parametrize("shape", [[(3,), (3,)]])
```

Reply:
output shape mismatched, add torch.atleast_2d and passed now